pages/_app.tsx

```diff
@@ -6,17 +6,26 @@ import Notifications from '../components/Notification'
 import NavBar from '../components/NavBar'
 import PageBodyContainer from '../components/PageBodyContainer'
 import useHydrateStore from '../hooks/useHydrateStore'
+import useRealm from '../hooks/useRealm'
+import { getResourcePathPart } from '../tools/core/resources'
 
 function App({ Component, pageProps }) {
   useHydrateStore()
   useWallet()
 
-  const title = 'Mango DAO'
-  const description =
-    'Discuss and vote on Mango DAO proposals. Join us in building Mango, the protocol for permissionless leverage trading & lending.'
-  const keywords =
-    'Mango Markets, DAO, Governance, Serum, SRM, Serum DEX, DEFI, Decentralized Finance, Decentralised Finance, Crypto, ERC20, Ethereum, Decentralize, Solana, SOL, SPL, Cross-Chain, Trading, Fastest, Fast, SerumBTC, SerumUSD, SRM Tokens, SPL Tokens'
-  const baseUrl = 'https://token.mango.markets'
+  const { realm, realmInfo } = useRealm()
+
+  // TODO: Show solana/realms branding when on the home page
+  const realmName = realmInfo?.mainnetName ?? realm?.info.name
+
+  const title = realmName ? `${realmName} DAO` : 'DAO'
+  const description = `Discuss and vote on ${title} proposals.`
+
+  // Note: ?v=1 is added to the url to force favicon refresh. Without it browsers would cache the last used and won't change it for different realms
+  // https://stackoverflow.com/questions/2208933/how-do-i-force-a-favicon-refresh
+  const faviconUrl = realmName
+    ? `/realms/${getResourcePathPart(realmName)}/favicon.ico?v=1`
```

Comment:
pretty sure you need a `v=${Date.now()}`

Reply:
thx, fixed